src/libraries/System.Resources.ResourceManager/tests/ResourceManagerTests.cs

```diff
@@ -204,7 +204,10 @@ public static void UsingResourceSet()
         [Fact]
         public static void BaseName()
         {
-            var manager = new ResourceManager("System.Resources.Tests.Resources.TestResx", typeof(ResourceManagerTests).GetTypeInfo().Assembly);
+            var manager = new ResourceManager("System.Resources.Tests.Resources.TestResx", typeof(ResourceManagerTests).Assembly);
+            Assert.Equal("System.Resources.Tests.Resources.TestResx", manager.BaseName);
+
+            manager = new(typeof(System.Resources.Tests.Resources.TestResx));
```

Comment:
NIT: we only use the compressed new statement when the type can be inferred from the left side of the assignement which isn't the case here (even though it is two lines above).